### DISCUSSION TITLE
[Bug] 新标签页打开动态详情会错误显示互动抽奖

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#734] [bug]
### Bug 描述

在 首页-动态 查看无抽奖动态的内容时可以正确显示（无抽奖的动态不展示、包含抽奖的动态仅显示“互动抽奖”文本，无超链接），但打开详情后无抽奖动态会在正文开头错误显示 互动抽奖 超链接
已在官方移动客户端确认原动态没有抽奖
问题具有普遍性，任意动态均可复现

### 操作步骤

```plain text
1 打开应用
2 进入 首页-动态
3 点击任意动态右上角，选择 打开详情
4 动态详情页可见错误
```

### 期望结果

在没有互动抽奖的动态内正确隐藏功能入口

### 截图

官方客户端显示![Screenshot_20240731_203245_HD](https://github.com/user-attachments/assets/4c858c55-fbf5-4772-a71c-fb4b4ff9a395)

动态聚合显示
![1a5aefb3-cd8a-479e-a487-370577956d13](https://github.com/user-attachments/assets/ad820bb8-5a1d-4b8e-bd2d-ea2d0b49d520)

动态详情页显示
![d77a7f00-ec3f-4169-b6c5-b8a9f385009d](https://github.com/user-attachments/assets/1ae116eb-3cdb-4b50-9311-13b0b15abea0)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 24H2 (Build 26257.5000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

软件版本4.7.0.101
另：设置-个性化 页内的 选择应用主题 主标题错误显示为 优先视频编码。仅文字错误，实际功能正常。
![2d1f42af-654e-42fc-a638-0af18d9e1804](https://github.com/user-attachments/assets/a884083c-6d69-4fdd-abfe-7f81bf2afc7b)

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
感谢反馈，动态详情是因为用的旧版的接口暂时应该不会处理
